packages/composables/src/useWishlist.ts

```diff
@@ -23,9 +28,21 @@ export type UseWishlistReturn = {
    */
   items: ComputedRef<string[]>;
   /**
-   * Wishlist items count
+   * Current page number
+   */
+  getCurrentPage(): ComputedRef<number>;
+  /**
+   * total pages count
+   */
+  getTotalPagesCount(limit?: number): ComputedRef<number>;
```

Comment:
you shouldn't return computed property from a function.

Btw you already have `ref` called `limit` - you can expose it for use and change and operate in computed properties on that value


```suggestion
  totalPagesCount: ComputedRef<number>;
```